### PR TITLE
Addition of postgres-exporter-ib

### PR DESCRIPTION
**Pull request by kanavphull**

### CODE REVIEW

File: community_images/postgres-exporter/ironbank/dc_coverage.sh
Comment (AnmolVirdi) [inline community_images/postgres-exporter/ironbank/dc_coverage.sh:1]: Does this cover all possible functionality for Postgres exporter?

**Review by AnmolVirdi — CHANGES_REQUESTED** [review]

File: community_images/postgres-exporter/ironbank/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+services:
+  postgres:
+    image: postgres:latest
+    container_name: postgres
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: mysecretpassword
+      POSTGRES_DB: mydatabase
+    ports:
+      - "5432:5432"
```
[the remark this answers is not in the document]
Comment (kanavphull) [reply community_images/postgres-exporter/ironbank/docker-compose.yml:12]: Sure, will add that. We would have to fetch the dynamically assigned port in dc_coverage.sh

File: community_images/postgres-exporter/ironbank/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+version: '2'
+
+services:
+  postgres:
+    image: postgres:latest
```
[the remark this answers is not in the document]
Comment (kanavphull) [reply community_images/postgres-exporter/ironbank/docker-compose.yml]: tested and added ✅

File: community_images/postgres-exporter/ironbank/dc_coverage.sh
Comment (kanavphull) [reply community_images/postgres-exporter/ironbank/dc_coverage.sh:1]: yes. I have added some more flags in docker-compose.yml itself to exercise some more functionality.

File: community_images/postgres-exporter/ironbank/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      - '--collector.stat_statements'
+      - '--collector.stat_user_tables'
+      - '--collector.stat_wal_receiver'
+      - '--collector.statio_user_indexes'
+      - '--collector.statio_user_tables'
+      - '--collector.wal'
+      - '--collector.xlog_location'
+
+volumes:
+  postgres-data:
```
Comment (AnmolVirdi) [inline community_images/postgres-exporter/ironbank/docker-compose.yml:49]: Please set up local driver here. `driver: local`

Comment (kanavphull) [reply community_images/postgres-exporter/ironbank/docker-compose.yml:49]: sure

**Pull request merged**